eng/Dependencies.props

```diff
@@ -15,6 +15,7 @@ and are generated based on the last package release.
   <ItemGroup Label=".NET team dependencies">
     <LatestPackageReference Include="Microsoft.Azure.SignalR" />
     <LatestPackageReference Include="Microsoft.Css.Parser" />
+    <LatestPackageReference Include="Microsoft.CodeAnalysis.Analyzers" />
```

Comment:
Why is this line needed❔ I don't see anything about this package in eng/Versions.props. Nor can I see any references to the package.

Reply:
This was a dependency I was using before but it is not needed anymore. Will remove.